example/docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
       - SYS_ADMIN
     devices:
       - /dev/fuse
+    security_opt:  # May not be required on non-ubuntu-based systems.
```

Comment:
Wording here conflicts with wording below. Is it required on ubuntu distros or non-ubuntu distros?

I'd also make it clear that it's required on Ubuntu **host** machines.

I'd reword this to "required on ubuntu-based hosts" so it's more direct and clear.